cmd/proxy/actions/index.go

```diff
@@ -0,0 +1,48 @@
+package actions
+
+import (
+	"encoding/json"
+	"net/http"
+	"strconv"
+	"time"
+
+	"github.com/gomods/athens/pkg/errors"
+	"github.com/gomods/athens/pkg/index"
+)
+
+// indexHandler implements GET baseURL/index
+func indexHandler(index index.Indexer) http.HandlerFunc {
+	const op errors.Op = "actions.IndexHandler"
+	return func(w http.ResponseWriter, r *http.Request) {
+		var (
+			err   error
+			limit int
+			since time.Time
+		)
+		if limitStr := r.FormValue("limit"); limitStr != "" {
+			limit, err = strconv.Atoi(limitStr)
+			if err != nil {
+				http.Error(w, err.Error(), http.StatusBadRequest)
+				return
+			}
+		}
+		if sinceStr := r.FormValue("since"); sinceStr != "" {
+			since, err = time.Parse(time.RFC3339, sinceStr)
+		}
+		if limit <= 0 {
+			limit = 2000
```

Comment:
Do we want the default limit to be this high? Should the default be configurable?

Reply:
This default is modeled after the https://index.golang.org mentioned on its home page: 

>'limit': the maximum length of the returned list. Default = 2000, Max = 2000

So far, I don't think we need to make it configurable because of 2000 was a bit too high for someone, they can always just pass a query string like `?limit=500`

Reply:
Hmm, I'm on the fence. I was wondering if it was a spec-related thing. I think its fine for now.